hapi-test/src/test/java/ca/uhn/hl7v2/app/ConnectionHubTest.java

```diff
@@ -125,6 +125,75 @@ public void testConnectWithSocketFactory() throws HL7Exception, IOException {
 
 	}
 
+	@Test
+	public void testConnectWithTimeout_WithoutSocketFactory() throws Exception {

```

Comment:
minor - test should say what the expected result is

testcase_initialconditions_expectedoutcome

Reply:
Done